automation_tools/__init__.py

```diff
@@ -1341,12 +1347,19 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         'satellite6-beta': cdn_install,
         'satellite6-cdn': cdn_install,
         'satellite6-downstream': downstream_install,
-        'satellite6-repofile' : repofile_install,
-        'satellite6-activationkey' : ak_install,
+        'satellite6-repofile': repofile_install,
+        'satellite6-activationkey': ak_install,
         'satellite6-iso': iso_install,
         'satellite6-upstream': upstream_install,
     }
     distribution = distribution.lower()
+
+    # Make sure downstream_install only is called for sat6.1 and sat6.2, as
+    # repofile_install and ak_install can be passed only for Sat6.3.
+    if satellite_version in ('6.1', '6.2'):
+        if distribution in ('satellite6-repofile', 'satellite6-activationkey'):
```

Comment:
Instead of calling repofile_install and activationkey_install for satellite_version 6.1 and 6.2, I want it to default to downstream_install. Hence I think `distribution in` is fine.